services/gateway/sources/version3/mappings/dposDelegate.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
 	name: '=,string',
 	totalVotesReceived: '=,string',
 	selfVotes: '=,string',
-	voteWeight: '=,string',
+	voteWeight: 'delegateWeight,string',
```

Comment:
I would recommend to do this change during refactoring phase because it requires more attention to update at all places. This [issue](https://github.com/LiskHQ/lisk-service/issues/1282) is just to update the gateway mapping.